samples/highcharts/series-arcdiagram/inverted/demo.css

```diff
@@ -0,0 +1,5 @@
+#container {
+    height: 400px;
+    max-width: 800px;
```

Comment:
The inverted arc diagram demo doesn't look good on desktop because it is 800px wide, so most of the plot area is white unused space. I suggest we simply make the `max-width` smaller.

Another room for improvement in this demo is the data labels. Currently they cover the nodes, even though there is plenty of space. So we should align them to the left of the nodes, and also make sure the overlapping prevention algorithm is not too aggressive (try setting a smaller `padding` on the data labels).